### DISCUSSION TITLE
Submitting a PR

### DISCUSSION THREAD

**Discussion by Daanvdplas** [smol-dot/smoldot#1267] [All]
I want to submit a PR for an additional example (connecting to a parachain). However, I'm facing an error that I don't have the permission / access rights.

**Comment by Daanvdplas** [comment]
The reason for me wanting to add this example is that I have been trying to use subxt with smoldot to connect to a parachain, but with the current api of subxt this is not possible. An example how it would look like to connect to a parachain might therefore be handy (and in general a nice addition).

**Comment by tomaka** [comment]
There's no need for access rights if you're opening the PR from a fork.
I don't do the "everyone pushes branches to the repository" thing that Parity does in this repository.
